### PR TITLE
Release 0.41.0.3

### PR DESCRIPTION
**Pull request by itaiad200**
Logo to be visible on Pypi and Astro registry.

### CODE REVIEW

File: .github/workflows/provider.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
           cp ./lakefs_provider/example_dags/lakefs-dag.py astro/dags/
 
       - name: Insert Dockerfile pip install
-        run: printf "\nRUN pip install --user dist/airflow_provider_lakefs-0.41.0.1-py3-none-any.whl" >> astro/Dockerfile
+        run: printf "\nRUN pip install --user dist/airflow_provider_lakefs-0.41.0.3-py3-none-any.whl" >> astro/Dockerfile
```
Comment (nopcoder) [inline .github/workflows/provider.yaml:41]: why don't we pass the version as build arg? we need a final static Dockerfile, or just push the result?

**Review by nopcoder — APPROVED** [review]
just comment for more information

File: .github/workflows/provider.yaml (hunk 1 above)
Comment (itaiad200) [reply .github/workflows/provider.yaml:41]: Sorry for pushing and missing this comment.
It's determined in the `setup.py` file, where version isn't injected too.
I'll change it to inject to both in a new PR.

Comment (itaiad200) [reply .github/workflows/provider.yaml:41]: Wait, I can't really add env var to workflows. Any other dynamic way for doing so??

**Pull request merged**